Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Tools/Notifications.hs

```diff
@@ -859,6 +859,37 @@ notifyStopModification person entityData = do
           [ if entityData.isEdit then "Customer edited stop!" else "Customer added a stop!"
           ]
 
+notifyDirectCallOnCallServiceDown ::
+  ( CacheFlow m r,
+    EsqDBFlow m r
+  ) =>
+  Id DMOC.MerchantOperatingCity ->
+  Id Person ->
+  Maybe FCM.FCMRecipientToken ->
+  m ()
+notifyDirectCallOnCallServiceDown merchantOpCityId personId mbDeviceToken = do
+  transporterConfig <- findByMerchantOpCityId merchantOpCityId (Just (DriverId (cast personId))) >>= fromMaybeM (TransporterConfigNotFound merchantOpCityId.getId)
```

Comment:
use `notifyDriverOnEvents` instead of creating this new function, also this title and body can be configurable in `merchantPushNotification` table